src/components/PetitionList/PetitionList.tsx

```diff
@@ -23,9 +54,13 @@ function PetitionList() {
           <span className={cx('end')}>청원마감일</span>
           <span className={cx('agree-num')}>참여인원</span>
         </div>
-        {mockData.map(petition => {
+        {contents.map(petition => {
           return <PetitionCard key={petition.id} petition={petition} />;
         })}
+        <div
+          className="observer"
+          ref={ref => setObserverRef(ref as HTMLDivElement)}
+        />
```

Comment:
```suggestion
        <div className="observer" ref={observerRef} />
```
위에 리뷰에 이어서 이렇게 변경하면 될것같습니다 여기서 한가지 첨언하고싶은건 대단한 내용은 아니지만 인터섹션 핸들러 함수내에서 `setItemIndex(prev => prev + 5);`이런식으로 값을 업데이트 시킬수 있습니다. 여기서 `setItemIndex(itenIndex + 5);`와 같이 업데이트 했다면 useEffect의 의존성에 itemIndex가 들어갈테고 스크롤시 추가로딩이 될때마다 새로운 IntersectionObserver객체가 생성되었을텐데 `setItemIndex(prev => prev + 5);` 이렇게 사용하면 그걸 막을수 있습니다

Reply:
그렇군요... 감사합니다!! 아직 공부할 내용이 많군욤